src/vendorcode/dasharo/uefi_capsules.c

```diff
@@ -0,0 +1,769 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
@krystian-hebel @SergiiDmytruk Does it have to be dasharo-specific? Cannot it go into mainline coreboot?

Reply:
It doesn't.  I just didn't know that upstreaming was an explicit goal here.  Will move the file to `src/lib` if won't find any better place.

Reply:
That was the main goal of the proposal: https://docs.dasharo.com/projects/capsule-updates/

We need to be in sync with the communities and establish a way to move forward since the very beginning, to ensure the whole process would go as smooth as possible.

Reply:
Made it more suitable for upstream: https://github.com/Dasharo/coreboot/compare/0ca09834457cc4c80472f049ad7d77d58b12f01a..1dbd00641ef16e4777f848694ec2c3131442e9a9